src/main/java/org/team1540/robot2019/datastructures/utils/UnitsUtils.java

```diff
@@ -0,0 +1,16 @@
+package org.team1540.robot2019.datastructures.utils;
+
+public class UnitsUtils {
+
+    private static final double INCHES_PER_METER = 39.3701;
+    private static final double INCHES_PER_FOOT = 12;
+    private static final double DECISECONDS_PER_SECOND = 10;
```

Comment:
You really do not need a constant for this 

Also you don't appear to be using it here?

Reply:
Not used rn, however, one should really never have constants (even simple ones). Instead, use functions to convert between units (remember how you mix up *10 and /10?).

Reply:
Alright. It's your preference